src/Compatibility/Core/src/Windows/FontNamedSizeService.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Microsoft.Maui.Controls.Internals;
+using Microsoft.Maui.Controls.Platform;
+
+[assembly: Microsoft.Maui.Controls.Dependency(typeof(Microsoft.Maui.Controls.Compatibility.Platform.UWP.FontNamedSizeService))]
```

Comment:
Were we trying to avoid assembly attributes for dependency declarations?

Reply:
We are, but for tests, this is how it is still used. For normal apps, it is registered and locked as part of the build with an explicit registration.

I am doing this now so I can reduce the churn on the test code. In another PR I will remove these attributes altogether and have each test register the required services.

But in order for that to work, we need to get rid of the Device's reliance on IPlatformServices. Also, this is just living in compatibility.